scripts/run-jenkins.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash -xe
+
+TIMEOUTCMD=scripts/babysitter
+export MONO_BABYSITTER_LOG_FILE=babysitter_report.json_lines
+
+if [[ ${label} == debian-arm* ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; fi
+if [[ ${label} == 'osx-i386' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no --build=i386-apple-darwin11.2.0"; fi
+if [[ ${label} == 'osx-amd64' ]]; then export MONO_TEST_RABBITMQ_SERVER=192.168.100.12; EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no"; fi
```

Comment:
Nice theory, except... we care about the Windows case here, and due to MAX_PATH on Windows, we use single-character job names for those, i.e. "Mono test build (pull requests) - i386 Windows" is `w` not `test-mono-pull-request-win32`

So label plus ghprb env var would work, your idea wouldn't